models/issue.go

```diff
@@ -1209,7 +1209,10 @@ func GetParticipantsByIssueID(issueID int64) ([]*User, error) {
 	userIDs := make([]int64, 0, 5)
 	if err := x.Table("comment").Cols("poster_id").
 		Where("issue_id = ?", issueID).
-		And("type = ?", CommentTypeComment).
+		And("`comment`.type = ?", CommentTypeComment).
+		And("`user`.is_active = ?", true).
+		And("`user`.prohibit_login = ?", false).
+		Join("LEFT", "user", "`user`.id = `comment`.poster_id").
```

Comment:
I'm not sure why we're doing a left join here. If the goal is to include comments that don't have a "valid" poster (e.g. deleted user), I don't think this will work; for those comments the `user.is_active` column will be NULL, and the `user.is_active = true` condition won't hold. And if we don't want to include such comments, why not just do an inner join?

Ditto elsewhere

Reply:
@ethantkoenig good point, I also think it should be changed to INNER join